web/src/beta/features/PluginPlayground/Plugins/presets/layers/add-geojson.ts

```diff
@@ -0,0 +1,109 @@
+import { FileType, PluginType } from "../../constants";
+
+const yamlFile: FileType = {
+  id: "layers-add-geojson-reearth-yml", 
+  title: "reearth.yml",
+  sourceCode: `id: layers-add-geojson
```

Comment:
```suggestion
  sourceCode: `id: layers-add-geojson-plugin
```
Better use different id for plugin & extension